examples/use-cases/SmartVF2SmartVF/README.md

```diff
@@ -0,0 +1,125 @@
+# Test Smart VF connection
```

Comment:
What is the difference with the `Kernel2Kernel` use case?

Reply:
I mean instead of name :)

Reply:
In SmartVF2SmartVF case, both Client and Endpoint pod are attached with smart VF (on top of this [PR](https://github.com/networkservicemesh/sdk-sriov/pull/179), but design approach for End point connection is under question (i.e. one VF per client connection) and discussing other possibilities [here](https://docs.google.com/document/d/10PC-291O4bKfYFQxVZnFquVcK8p5wDWchnAdxkODCLk/edit?usp=sharing)).

Reply:
@Bolodya1997 I'm not sure if MLX provides 10G NIC which has hw offload capability, we've used only 100G Connectx-5 NIC for testing. anyway this NIC is totally different from legacy SRIOV NIC and also different drivers to be bound on PFs and VFs. we can still use same sriov.conf file, but this would go into another `physicalFunctions`.